src/api/lib/tasks/databases.rake

```diff
@@ -97,6 +97,22 @@ namespace :db do
       end
       puts "Everything looks fine!"
     end
+
+    desc "Verify that structure.sql does not use any columns with type = bigint"
+    task verify_no_bigint: :environment do
+      puts 'Checking db/structure.sql for bigint'
+
+      bigint_lines = %x{grep "bigint" #{Rails.root}/db/structure.sql}
+
+      unless bigint_lines.blank?
+        abort <<-STR
```

Comment:
[nitpick] Have you tried that this will actually fail in travis?

Reply:
It will not...

Reply:
> It will...

:rofl: I like how this was edited from: 

> It will not...

But no I have not actually tested that this will fail in travis (I just copied the procedure from rake db:structure:verify). I can open a DO NOT MERGE PR to test this actually fails if you guys would like me to?

Reply:
no need to test `abort` :)